### ISSUE TITLE
Fatal error decrypting master key!

### ISSUE THREAD

**Issue by gbschenkel** [open-keychain/open-keychain#2082]
I am trying to change the password from my private key.

## Expected Behavior
The password been changed.

## Current Behavior
[START] Performing key operation
 [DEBUG] Fetching key to modify (xxxx)
    [ERROR] Fatal error decrypting master key! This is likely a programming error, please file a bug report!

## Steps to Reproduce (for bugs)
1. Open your key
2. Go to menu(3 dots)
3. Press Change Password
4. Fill with the new password.

## Context
I was trying to import my private key to use with GnuPG/Enigmail. I couldn't import because everytime it say my password is wrong, them I tried to change it on OpenKeychain and bug happen. This key I created few days ago on OpenKeychain using the Security Token option with my YubiKey NEO.

## Your Environment
* Android Version: LineageOS 7.1.1
* OpenKeychain Version: 4.2.4
* From Google Play or F-Droid?: F-Droid

**Comment by Valodim** [comment]
This is indeed a bug. I'm not sure if we ever supported changing yubikey pins, but changing the "password" of such a key makes no sense. Did I read it right that the 'Backup key' feature didn't work for your key? Does it ask for a password?

Also, can you please add a screenshot of the 'Subkeys' tab of your key (under 'Advanced'?)

**Comment by gbschenkel** [comment]
Yep, I tried to import, and won't work on GnuPG Manager, It say: "Error - The first block OpenPGP isn't public key block."
![image](https://cloud.githubusercontent.com/assets/1715069/24203253/f998f00a-0ef3-11e7-96fb-9aa002c739da.png)

Then I tried on the console with this command:
gpg --decrypt .\backup_2017-03-21.sec.pgp | gpg --import .\backup_2017-03-21.sec.pgp
Ask me the password but refuse it. (I found this command in some place)
![image](https://cloud.githubusercontent.com/assets/1715069/24203345/38ef5bcc-0ef4-11e7-81c7-343b7bff0bf5.png)

##Your requested screenshot
![image](https://cloud.githubusercontent.com/assets/1715069/24203626/093df9fa-0ef5-11e7-8a1b-c4520add1acd.png)

**Comment by Valodim** [comment]
your import command is wrong. please try `gpg --decrypt .\backup_2017-03-21.sec.pgp | gpg --import`

**Comment by gbschenkel** [comment]
Same thing....
I was checking others things, and when I try to change password, the OpenKeychain ask my PIN but don't ask to tap my YubiKey NEO on it. If I try to do a simple text encryption, it ask for the PIN+token and works, also sending email. Maybe the fact of not asking for the token after put the PIN, when I try change the password, could be the problem.
